test_data.py

```
@@ -0,0 +1,94 @@
+def generate_random_mg_id() -> str:
+    return f"MG {generate_random_string(3)}"
+
+def hash_mg_id(mg_id: str) -> str:
+    return hashlib.sha256(mg_id.encode()).hexdigest()
+
+def create_MG_HASH_MAP_table(con,num_mg):
+    con.sql(f"drop table if exists {TABLE_NAMES['MG_HASH_MAP']}")
+    con.sql(f"CREATE TABLE if not exists {TABLE_NAMES['MG_HASH_MAP']} (mg_id TEXT PRIMARY KEY, hash TEXT)")
+    con.sql(f"DELETE FROM {TABLE_NAMES['MG_HASH_MAP']}")
+    
+    for i in range(num_mg):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)